Headers/DebugServer2/Target/Windows/Process.h

```diff
@@ -59,6 +59,7 @@ class Process : public ds2::Target::ProcessBase,
   }
 
 public:
+  ErrorCode getFileLoadAddress(std::string const &file_path, Address &address);
```

Comment:
Instead of having `getFileLoadAddress` you can have something like `enumerateMappedFiles` or `enumerateLoadedFiles`, that would take a callback just like `enumerateSharedLibraries`. Then you can call `enumerateLoadedFiles` from `DebugSessionImpl` and put whatever logic for finding mapped files there (e.g.: taking only the basename, etc).